.github/workflows/main.yml

```diff
@@ -50,11 +50,14 @@ jobs:
         include:
           - python: "3.9"
             image_tag: "nightly"
-            from_tag: "latest"
-            container_file: "Containerfile.source"
+          - python: "3.8"
+            pulp_cli: "0.17.1"
+            image_tag: "3.22"
+            from_tag: "3.22"
+            container_file: "Containerfile"
```

Comment:
@mdellweg, I had to add `Containerfile` back because we do not release pulp_ostree in stable images.